frontend/src/booking/calendar.vue

```diff
@@ -177,6 +178,11 @@
         'showExamInventoryModal',
         'showSchedulingIndicator',
       ]),
+      config() {
```

Comment:
What is the purpose of this? It looks like we could just add:

`selectable: true`

On the initial setup object in `data ()`. Thoughts?

Reply:
I actually tried that initially, and the calendar still wasn't performing. After consulting scott on friday morning, he came to the suggestion of making that config function, to which I was actually able to get the calendar to be selectable, and able to insert events into it.